src/test/validation_chainstatemanager_tests.cpp

```diff
@@ -769,4 +770,44 @@ BOOST_FIXTURE_TEST_CASE(chainstatemanager_snapshot_completion_hash_mismatch, Sna
     }
 }
 
+/** Helper function to parse args into args_man and return the result of applying them to opts */
+util::Result<void> SetOptsFromArgs(ArgsManager& args_man, ChainstateManager::Options& opts,
+                                   const std::vector<const char*>& args)
+{
+    std::vector<const char*> argv = {"ignore"};
+    argv.insert(argv.end(), args.begin(), args.end());
+
+    std::string error{};
+    BOOST_REQUIRE(args_man.ParseParameters(argv.size(), argv.data(), error));
```

Comment:
nit:
```suggestion
    BOOST_REQUIRE(args_man.ParseParameters(argv.size(), argv.data(), error));
    BOOST_REQUIRE(error.empty());
```

Reply:
This feels like a distraction to me tbh, we're not unit testing `ParseParameters` here. I think it's safe to rely on `ParseParameters` returning `false` when there's an error.